modules/sql-engine/src/integrationTest/sql/types/string/test_unicode_escapes_in_literals.test_ignored

```diff
@@ -0,0 +1,62 @@
+# name: test/sql/types/string/test_unicode_escapes_on_literals.test
+# description: SQL feature F393 (Unicode escapes in literals)
+# group: [string]
+# Ignore: https://issues.apache.org/jira/browse/IGNITE-22221
+
+# insert unicode strings into the database
+statement ok
+CREATE TABLE t(id INTEGER, s VARCHAR);
+
+statement ok
+INSERT INTO t VALUES (1, U&'d\\0061t\\+000061'), (2, N'd\\0061t\\+000061'), (3, 'd\\x61t\\x0061'), (4, 'd\\u0061t\\U00000061')
```

Comment:
I don't fully understand why do you use such values.
For the first string it should be `U&'d\0061t\0061'` - in this case we will have another error and seems which is right.
for 2nd - it's national characters - separate feature
for 3rd and 4th - what reason to check it?